feeds/us-ca.json

```diff
@@ -90,6 +90,30 @@
             "type": "transitland-atlas",
             "transitland-atlas-id": "f-9q5-metro~losangeles~rail"
         },
+        {
+            "name": "Antelope-Valley-Transit-Authority",
+            "type": "transitland-atlas",
+            "transitland-atlas-id": "f-antelope~valley~transit~authority"
+        },
+        {
+            "name": "Glendale-Beeline",
+            "type": "transitland-atlas",
+            "transitland-atlas-id": "f-9q5f-glendalebeeline"
```

Comment:
It looks like this gives a 403 (Permission denied).
Maybe it needs an addional authentication header with an API key, or something like that?

Reply:
it seems to allow download when the feed url is opened in firefox. I can't seem to find information about an API key or required headers (or any info other than the transitland entries) so maybe it makes sense to remove Beeline for now.

Reply:
The server only seems to answer with the data if the client advertises support for compressed transfers. Apparently we don't do that right now, we should fix that.
If you don't want to do it right now, I suggest adding skip: true with a skip-reason for now so we can add it later.

Reply:
Yeah, it's marked as skipped for now.